### ISSUE TITLE
ubuntu 16.04 install

### ISSUE THREAD

**Issue by RoDuth** [Ghini/ghini.desktop#143]
Not sure if this is related to issue #141 but thought I would just raise it so you know.
Installing Ghini on my new laptop with a fresh Ubuntu install.
`ctrl`+`alt`+`t`
`$ cd Downloads`
`$ wget https://raw.githubusercontent.com/Ghini/ghini.desktop/master/scripts/devinstall.sh`
`$ chmod +x devinstall.sh`
`$ ./devinstall.sh`
(My usual method for a fresh install)
then noticed a few things didn't seem right e.g.
```
Building lxml version 3.6.0.
Building without Cython.
Using build configuration of libxslt 1.1.28
/usr/bin/ld: cannot find -lz
collect2: error: ld returned 1 exit status
error: Setup script exited with error: command 'x86_64-linux-gnu-gcc' failed with exit status 1
```
then when I tried to run Ghini I got this
`pkg_resources.DistributionNotFound: The 'lxml' distribution was not found and is required by bauble`
ran
`$ sudo apt-get install python-lxml`
re-ran 
`$ ./devinstall.sh`
which seemed to fix it.
Just thought I'd let you know about a very minor issue.

**Comment by mfrasca** [comment]
unrelated to #141, it is probably associated to different packaging of libraries in new distribution.
if it works for you, it might work for others, but I think we should solve the initial report from the installation script, not the report from python.
the installation script tries to compile a library and does not manage (see [this Q&A](http://stackoverflow.com/questions/3373995/usr-bin-ld-cannot-find-lz) as well as [this one](http://askubuntu.com/questions/716945/usr-bin-ld-cannot-find-lz)). do you think you could remove the python library you installed, then install the development library for zlib, then re-run the installation script?

the solution according to me is to add a test in the devinstall script that checks that the development libraries for zlib are in place, and that it suggests a solution if they are not found.

**Comment by RoDuth** [comment]
sorry @mfrasca I never read this and hence never did what you suggested...When I get a little time I'll give it a go, maybe later today.  I'm assuming you mean `sudo apt-get remove python-lxml` > `audo apt-get install zlib-devel` > `./devinstall.sh` ? 

In the meantime, I did find another Ubuntu 16.04 issue that I had been largely ignoring but finally lost patience with so decided to look into it.  I found a simple solution and so am posting it here for anyone else with a similar problem.  

The issue was: Ghini seemed to be unable to "open" anything. i.e. clicking on "Search Google" did nothing and reports gave a message saying you had to go find them in the /tmp folder.  It turns out that Ghini uses `gnome-open` to open files when a gnome desktop environment is detected. According to [this](http://askubuntu.com/questions/68961/what-can-i-use-instead-of-gnome-open) `gnome-open` has been superseded by `gvfs-open` and in Ubuntu the more desirable approach is to use `xdg-open` both of which I already had on my system.  It looks like the [python desktop package](https://pypi.python.org/pypi/desktop?) is the cause and even though there seems to be a latter version than what is included in Ghini it's no different.

So, the simple solution for anyone currently wanting to install Ghini on a ubuntu 16.04 system... 
install libgnome2-bin.

`$ sudo apt install libgnome2-bin`

I just hope it doesn't cause any conflicts! I guess another approach could be to make an executable link? something like `/usr/bin/gvfs-open` > `~/bin/gnome-open`.  Or is it possible to change `gnome-open` to `gvfs-open` in /bauble/utils/desktop.py without causing harm to any other users?  How long ago did `gvfs-open` start getting distributed with gnome systems?
